src/transformers/models/layoutlm/modeling_layoutlm.py

```diff
@@ -733,19 +734,17 @@ class PreTrainedModel
     @replace_return_docstrings(output_type=BaseModelOutputWithPoolingAndCrossAttentions, config_class=_CONFIG_FOR_DOC)
     def forward(
         self,
-        input_ids=None,
-        bbox=None,
-        attention_mask=None,
-        token_type_ids=None,
-        position_ids=None,
-        head_mask=None,
-        inputs_embeds=None,
-        encoder_hidden_states=None,
-        encoder_attention_mask=None,
```

Comment:
This one is unused too

Reply:
I think it's out of scope of this PR, but good catch nonetheless! I'm not sure why the arguments are there if they're not used, but you can raise an issue after this PR is merged if you like.